homeworks/olha.tymchyshyn_olyatmchshn/PokemonProject/login.html

```diff
@@ -52,9 +52,12 @@ <h1>Login page</h1>
                         <br>
                     </div>
 
-                    <input type="submit" value="Login">
+                    <div id="errors-container"></div>
```

Comment:
Please correct the styles of input text 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/34cfd69c-f063-4ec0-9548-f053a3433a3f)

Reply:
Did you [switch off the autocomplete](https://github.com/MastersAcademy/fe-course-basic-2023/pull/298/files#diff-31d3029c14f9372bd99eabfbcd21b0caccf69e5261e8fd2df93c2be1ae804c83R41) for some specific reason? If not, please switch on back, it is very useful from the UX perpective.
> <input type="email" id="email" name="email" **autocomplete="off"** placeholder="Enter your email address" required>